source/reference/api/start.txt

```diff
@@ -111,6 +127,64 @@ Request Body Parameters
 
        Default value is ``false``.
 
+
+.. _c2c-api-start-sharding:
+
+Sharding Parameters
+~~~~~~~~~~~~~~~~~~~
+
+To synchronize from a replica set to a sharded cluster, set the 
+``sharding`` option to shard collections on the destination cluster.
+
+The ``sharding`` option has the following parameters:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :widths: 15 15 70
+
+   * - Parameter
+     - Type
+     - Description
+
+   * - ``createSupportedIndexes``
+     - boolean
+     - Optional. Sets whether synchronization creates a supporting index 
+       for the shard key, if none exists.  Defaults to ``false``.
+
+   * - ``shardingEntries``
+     - array of documents
+     - Required. Sets the namespace and key of collections to shard 
+       during synchronization.
+
+       Any collections not included in this array synchronize to unsharded 
+       collections on the destination cluster.
+
+   * - | ``shardingEntries``
+       | ``.collection``
+     - string
+     - Sets the collection to shard. 
+
+   * - | ``shardingEntries``
+       | ``.database``
+     - string
+     - Sets the database of the collection to shard.
+ 
+   * - | ``shardingEntries``
+       | ``.shardCollection``
+     - document 
+     - Sets the shard key to generate on the destination cluster.
+
+   * - | ``shardingEntries``
+       | ``.shardCollection``
+       | ``.key``
+     - array 
+     - Sets the indexes to use for the shard key. 
```

Comment:
I think it should say "Sets the **fields** to use for the shard key.". Also it might be too long for here but should it say somewhere that the rules are the same as for shardCollection (in term of supporting up to one `"hashed"` field and 0-N `"1"` range fields) ?

Reply:
Fixed